lib/internal/Magento/Framework/Stdlib/DateTime/Filter/DateTime.php

```diff
@@ -42,7 +42,7 @@ public function __construct(\Magento\Framework\Stdlib\DateTime\TimezoneInterface
     public function filter($value)
     {
         try {
-            $dateTime = $this->_localeDate->date($value, null, false);
+            $dateTime = $this->_localeDate->date($value, null, false, false);
```

Comment:
Such check was already proposed and does not seem to be valid, please check https://github.com/magento/magento2/pull/11156/files#r142015395 and apply appropriate changes.

Reply:
Hi @fernandofauth, are you going to continue work on this PR?